MaineC/mainec.github.io
# Add Hugo, hugo-book theme, customisation, and examples

This is the very first commit to the hugo-fied Inductive Bias blog. The commit adds inital hugo config as well as the "hugo-book" theme. A custom css in `assets/_custom.scss` ensures the header image is visible. A custom index list page in `layouts/list.html` shows blog post snippets by default, and adds support for a "excludefromsnippet" page property to exclude pages from snippet generation - this is e.g. used in the `about.md` page.

How to test:
- Check out the branch / pr
- Run `hugo server`
- Point browser to http://localhost:1313
